virttest/utils_windows/virtio_win.py

```diff
@@ -0,0 +1,104 @@
+"""
+Windows virtio-win utilities
+"""
+
+import re
+
+from . import drive
+from . import system
+
+
+ARCH_MAP_ISO = {"32-bit": "x86", "64-bit": "amd64"}
+ARCH_MAP_VFD = {"32-bit": "i386", "64-bit": "amd64"}
+
+
+def arch_dirname_iso(session):
+    """
+    Get architecture directory's name - iso media version.
+
+    :param session: Session object.
+
+    :return: Directory's name.
+    """
+    return ARCH_MAP_ISO.get(system.os_arch(session))
+
+
+def arch_dirname_vfd(session):
+    """
+    Get architecture directory's name - vfd media version.
+
+    :param session: Session object.
+
+    :return: Directory's name.
+    """
+    return ARCH_MAP_VFD.get(system.os_arch(session))
+
+
+def _product_info(session):
+    match = re.search(r"Windows((?: )Serverr?)? (\S+)(?: (R2))?",
```

Comment:
is `Serverr` a typo?

Reply:
No, some windows system would present the trailing `r` as the registered trademark ® in this field, so the above regex must have this pattern (`r?`).

Reply:
better to comments in your code.

Reply:
Sure, thanks.